### ISSUE TITLE
Log In FAILED

### ISSUE THREAD

**Issue by willhoo** [finndev/PokeBuddy#804]
I cant log in,keep login failed!   anyone facing a same problem with me?

**Comment by curtiola** [comment]
Which log in failed, pokebuddy or your account?

**Comment by matt2310** [comment]
mines sitting right here

[P - 22:49:04] Logging in with method: PTC as Auth Provider

**Comment by curtiola** [comment]
It might be PTC servers, have you tried logging in on your mobile with location turned off? (Off to avoid soft ban)

**Comment by matt2310** [comment]
i cant log out if locations are off, it just says gps signal not found

**Comment by curtiola** [comment]
You can log in with it off, just to see if you can connect in general, if you make it to your inventory and see gps signal not found on the same account or whatever it's probably not ptc servers

**Comment by matt2310** [comment]
im logged in on another account, im using vpn on my computer for this

**Comment by matt2310** [comment]
damn its working now, thank god

**Comment by matt2310** [comment]
so how do i get off with locations off? cuz as soon as i take it off it says gps not located and i cant click settings

**Comment by curtiola** [comment]
Close the app with the task manager, then next time you open the app, as soon as you see Niantic (before gyarados loading screen) press the home button and wait like 30 seconds, then when you open it again you can press sign out above the loading bar

**Comment by matt2310** [comment]
OK thanks

**Comment by matt2310** [comment]
so i could essentially use that method to favorite pokemon i want to keep right? without pinging my location

**Comment by curtiola** [comment]
Yeah that's right, some people say favourite genre works to stop transfer and some say it doesn't, I haven't tried, but make sure you have an extra bulbasaur or charmander marked as favourite to test it, one of the low pokemon id #s

**Comment by curtiola** [comment]
Yeah that's right, some people say favourite genre works to stop transfer and some say it doesn't, I haven't tried, but make sure you have an extra bulbasaur or charmander marked as favourite to test it, one of the low pokemon id #s
